eloquent/distribution.yaml

```diff
@@ -969,19 +969,11 @@ repositories:
       version: eloquent
     status: developed
   libg2o:
-    doc:
```

Comment:
Did you mean to remove the doc and source entries for this package?

Reply:
yes, since it refers to an external third-party, non-ament, package. The buildfarm fails in this case.